docs/.project/RELEASE_TEMPLATE.md

```diff
@@ -48,6 +48,7 @@ _To be done after code freeze to prepare and test the release._
 * [ ] compile a list of blog worthy changes as input to [release blog](https://confluence.camunda.com/pages/viewpage.action?pageId=178590449)
 * [ ] merge to main: `git checkout main && git merge develop`
 * [ ] create release candidate (`npm run release:rc -- [preminor|premajor|prepatch]`), cf. [release schema](https://github.com/bpmn-io/internal-docs/blob/main/releases/RELEASE_SCHEMA.md)
+  * [ ] to create further release candidates after the first one use `npm run release:rc -- prerelease`
   * [ ] wait for [release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
```

Comment:
```suggestion
  * [ ] wait for [release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
  * [ ] (optional) create further release candidates after the first one use `npm run release:rc -- prerelease`
```

Reply:
We can but don't have to create further release candidates, so they are optional. With this change, I think it's ready to merge.